lib/vanagon/platform/windows.rb

```diff
@@ -107,7 +107,11 @@ def generate_msi_package(project)
         target_dir = project.repo ? output_dir(project.repo) : output_dir
         cg_name = "ProductComponentGroup"
         dir_ref = "INSTALLDIR"
-        candle_flags =  "-dPlatform=#{@architecture} -arch #{@architecture} -ext WiXUtilExtension -ext WixUIExtension"
+        wix_extensions =  "-ext WiXUtilExtension -ext WixUIExtension"
+        candle_flags =  "-dPlatform=#{@architecture} -arch #{@architecture} #{wix_extensions}"
```

Comment:
Will put this up as a separate ticket [(RE-6525)](https://tickets.puppetlabs.com/browse/RE-6525) as I need to investigate if there are any other implications, but it looks like a good idea.